keyboards/westm/westm68/keymaps/via/keymap.c

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2021 WestM
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT_65_ansi( /* Base */
```

Comment:
This needs to have 4 layers explicitly defined here.

Also, please replace the tabs with spaces.

Reply:
Hi Drashna, thanks for pointing on the other changes. I'm not exactly sure for this section what you mean by 4 layers? As in it should have 4 layers of other keymaps? Thanks!

Reply:
It needs 4 layers defined/filled out in the via keymap.  You only have 2.  If you don't fill out all of the layers needed, they can/will be filled with random/junk data in VIA

Reply:
Okie! Added in the extra layers.